gradle.properties

```diff
@@ -1 +1,4 @@
 org.gradle.jvmargs=-Dfile.encoding=UTF-8
+
+dokkaVersion=0.9.+
+kotlinVersion=1.0.+
```

Comment:
You need to specify extact versions. No dynamic selectors, please.
Also, we already have a standard of defining library versions for the project here: https://github.com/ratpack/ratpack/blob/master/gradle/dependencies.gradle